operator/src/test/java/org/keycloak/operator/utils/K8sUtils.java

```diff
@@ -93,7 +92,7 @@ public static String inClusterCurl(KubernetesClient k8sclient, String namespace,
                             .build())
                     .done();
             Log.info("Waiting for curl Pod to finish running");
-            Awaitility.await().atMost(2, MINUTES)
+            Awaitility.await().atMost(2, TimeUnit.MINUTES)
```

Comment:
Not sure why to make this change but ok. 😉

Reply:
Just made imports uniform